app/configurations/config.hsl.js

```diff
@@ -275,27 +275,41 @@ export default {
 
   fareMapping: {
     'HSL:hki': 'HSL:hki',
+    'HSL:hki2': 'HSL:hki',
     'HSL:esp': 'HSL:esp',
+    'HSL:esp2': 'HSL:esp',
     'HSL:van': 'HSL:van',
+    'HSL:van2': 'HSL:van',
     'HSL:ker': 'HSL:ker',
     'HSL:kir': 'HSL:kir',
     'HSL:seu': 'HSL:seu',
     'HSL:seu2': 'HSL:seu',
     'HSL:seu3': 'HSL:seu',
     'HSL:seu4': 'HSL:seu',
+    'HSL:seu5': 'HSL:seu',
     'HSL:lse': 'HSL:lse',
     'HSL:lse2': 'HSL:lse',
     'HSL:lse3': 'HSL:lse',
     'HSL:lse4': 'HSL:lse',
     'HSL:lse5': 'HSL:lse',
+    'HSL:lse6': 'HSL:lse',
     'HSL:kse': 'HSL:kse',
+    'HSL:kse1': 'HSL:kse',
     'HSL:kse2': 'HSL:kse',
     'HSL:kse3': 'HSL:kse',
     'HSL:kse4': 'HSL:kse',
     'HSL:kse5': 'HSL:kse',
     'HSL:kse6': 'HSL:kse',
     'HSL:kse7': 'HSL:kse',
     'HSL:kse8': 'HSL:kse',
+    'HSL:kse9': 'HSL:kse',
+    'HSL:kse10': 'HSL:kse',
+    'HSL:kse11': 'HSL:kse',
+    'HSL:kse12': 'HSL:kse',
+    'HSL:kse13': 'HSL:kse',
+    'HSL:kse14': 'HSL:kse',
+    'HSL:kse15': 'HSL:kse',
+    'HSL:kse16': 'HSL:kse',
   },
 
```

Comment:
Would it be feasible to just drop the numeric part of each identifier (i.e. 'HSL:kse10' -> 'HSL:kse')? Then this fareMapping would not be required to exist at all. Any new numeric extensions in the future would also work right away.

It also seems that the value of each pair is used as a key to retrieve the name of the ticket. Perhaps a unit test to check that the name exists for all possibilities would be in order.

We can revisit these ideas once the fare zones will be updated for the HSL area.